webapp/tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": false,
+    "noEmit": true,
```

Comment:
would need noEmit to be false. Ensure that on each commit that changes to this file doesnt change

Reply:
Understood, thanks

Reply:
amended